marklogic-data-hub/src/main/resources/ml-modules/root/data-hub/5/impl/hub-es.sjs

```diff
@@ -0,0 +1,128 @@
+/**
+ Copyright 2012-2019 MarkLogic Corporation
+
+ Licensed under the Apache License, Version 2.0 (the "License");
+ you may not use this file except in compliance with the License.
+ You may obtain a copy of the License at
+
+ http://www.apache.org/licenses/LICENSE-2.0
+
+ Unless required by applicable law or agreed to in writing, software
+ distributed under the License is distributed on an "AS IS" BASIS,
+ WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ See the License for the specific language governing permissions and
+ limitations under the License.
+ */
+
+'use strict';
+const sem = require("/MarkLogic/semantics.xqy");
+
+function getPropertyRangePath(entityIRI, propertyPath) {
```

Comment:
@ryanjdew I think this is fine here for 5.2.0, I'm thinking we can combine this with entity-lib.sjs in my DHFPROD-4379 PR for 5.3.0.